units/src/amount/unsigned.rs

```diff
@@ -134,13 +134,13 @@ impl Amount {
     /// Converts from a value expressing a whole number of bitcoin to an [`Amount`].
     #[allow(clippy::missing_panics_doc)]
     pub fn from_int_btc<T: Into<u16>>(whole_bitcoin: T) -> Amount {
-        Amount::from_int_btc_const(whole_bitcoin.into())
+        Amount::from_btc_u16(whole_bitcoin.into())
     }
 
     /// Converts from a value expressing a whole number of bitcoin to an [`Amount`]
     /// in const context.
     #[allow(clippy::missing_panics_doc)]
-    pub const fn from_int_btc_const(whole_bitcoin: u16) -> Amount {
+    pub const fn from_btc_u16(whole_bitcoin: u16) -> Amount {
```

Comment:
Should this be u32?  The idea being that u32 max is greater than max number of bitcoins 21 million?  u16 is 65,535 which isn't enough to represent every bitcoin.

Reply:
Ah I see, this allows for a restricted bounds since u32 max would be `429496729500000000` sats which is greater than the maximum number sats.  Sorry, didn't think that through before.